packages/ui/src/primitives/tailwind/Select/index.tsx

```diff
@@ -396,7 +397,8 @@ const Select = ({
           }
         }}
       >
-        {filteredOptions.length > 0 ? (
+        {!isEmpty(filteredOptions) &&
```

Comment:
prefer not using lodash helpers for simple operations, like checking if an array is empty. why does `filteredOptions.length > 0` not suffice?